cmd/cloud/cluster.go

```diff
@@ -15,16 +15,16 @@ func init() {
 	clusterCmd.PersistentFlags().String("server", "http://localhost:8075", "The provisioning server whose API will be queried.")
 
 	clusterCreateCmd.Flags().String("provider", "aws", "Cloud provider hosting the cluster.")
-	clusterCreateCmd.Flags().String("size", "SizeAlef500", "The size constant describing the cluster.")
+	clusterCreateCmd.Flags().String("size", "SizeAlef500", "The size constant describing the cluster. Add '-HA2' or '-HA3' to the size for multiple master nodes.")
```

Comment:
Interesting choice to append it to the size

Reply:
Do you think this is a bad method @jwilander? I don't see any obvious issues here, and it seems pretty flexible, but I am open to concerns.

Reply:
FYI Gabe and I chatted about it on a call, I'm OK with it